guake/tests/test_about.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from guake import guake_version
+from guake.about import AboutDialog
+from locale import gettext as _
+
+
+class TestAboutDialog:
```

Comment:
Prefer the test_* method of unit test with pytest, so you can encapuslate pytest.fixture